src/blob/index/range.rs

```diff
@@ -58,3 +44,178 @@ impl RangeFilter {
         bincode::serialize(&self).map_err(|e| e.into())
     }
 }
+
+const STRUCT_NAME: &str = "RangeFilter";
```

Comment:
Why custom (De)serialization implementation is needed

Reply:
Changed to usage of `(de)serialize_with` attribute for key fields